FannyAndersson/PayWay
# Feature/profile component

Hello!
Test please user's profile, both read and update.
NOTE: 

- This commit brings changes in markup of ContextKeeper to wrap all components in one responsive container (Col), after merge another components need to be fixed with css grid classes, it is minor. 
- Property limit is added to mongoose User.model.
- User has no opportunity to delete account or change password from Profile now. These options are supposed to be another features.

How to test:

- Be sure you logged in
- Click on Settings in opened menu
- Try to update different fields - name, phone number, email, limit amount (limit to send), click on Save button
- Try to change phone number to number which belongs to another user, you will get an error
- Try to change email to email which belongs to another user, you will get an error
- If you changed name or phone number, click on Menu and check if name or phone number updated in the user's info (purple background)
